R/dash.R

```diff
@@ -112,16 +112,15 @@ Dash <- R6::R6Class(
                           server = fiery::Fire$new(),
                           static_folder = NULL,
                           serve_locally = TRUE,
-                          routes_pathname_prefix = '/',
-                          requests_pathname_prefix = '/',
-                          suppress_callback_exceptions = FALSE) {
+                          routes_pathname_prefix = NULL,
+                          requests_pathname_prefix = NULL,
+                          suppress_callback_exceptions = FALSE,
+                          components_cache_max_age = 2678400) {
```

Comment:
🔪 `cache_max_age` - As we discussed today, this will be handled via the query strings and the cache will be infinite.

Reply:
OK, got it. Just curious, do these lines in `dash.py` remain for backward compatibility, then?

https://github.com/plotly/dash/blob/2a4c63ed6215973ca34262c61d6ce25947cf3643/dash/dash.py#L95

https://github.com/plotly/dash/blob/2a4c63ed6215973ca34262c61d6ce25947cf3643/dash/dash.py#L140

https://github.com/plotly/dash/blob/2a4c63ed6215973ca34262c61d6ce25947cf3643/dash/dash.py#L514

I was trying to reimplement this logic in R, down to the value of `2678400`.

Reply:
Ah I see. Looks like this was added in https://github.com/plotly/dash/pull/387/. I'm not actually sure what the use case is. But sure, let's keep it in just for parity 👍

Reply:
FWIW 2678400 is 31 days, so functionally equivalent to infinite most of the time.